proto/axelar/nexus/v1beta1/events.proto

```diff
@@ -43,3 +43,8 @@ message RateLimitUpdated {
   google.protobuf.Duration window = 3
       [ (gogoproto.stdduration) = true, (gogoproto.nullable) = false ];
 }
+
+message NewGeneralMessage {
+  string id =1 [(gogoproto.customname) = "ID"];
+  bytes payload_hash = 2;
+}
```

Comment:
new line

Reply:
so proud that it's not just me anymore complaining about missing new line now lol.